user/templates/user/home.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'user/base.html'%}
```

Comment:
```suggestion
{% extends 'user/base.html' %}
```
細かいところですが、タグの最後にも１スペース入れるようにしましょう。`templates`ディレクトリの中のファイルで、このファイルを含めて他にも同じ修正をする箇所があるので、探してみてください。

Reply:
確かに入れないとダメでした！修正しました。